packages/components/src/tabs/index.tsx

```diff
@@ -0,0 +1,166 @@
+/**
+ * External dependencies
+ */
+import * as Ariakit from '@ariakit/react';
```

Comment:
We'll have to add `// eslint-disable-next-line no-restricted-imports` above all ariakit imports